code/game/objects/random/loot.dm

```diff
@@ -335,30 +335,40 @@
 		/obj/item/storage/pill_bottle/mortaphenyl = 3,
 		/obj/item/storage/pill_bottle/happy = 2,
 		/obj/item/storage/pill_bottle/zoom = 2,
+		/obj/item/reagent_containers/pill/cocaine = 3,
+		/obj/item/reagent_containers/pill/heroin = 2,
+		/obj/item/reagent_containers/hypospray/autoinjector/stimpack = 2,
+		/obj/item/reagent_containers/hypospray/autoinjector/berserk = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/impedrezene = 0.4,
+		/obj/item/reagent_containers/hypospray/autoinjector/night_juice = 0.3,
+		/obj/item/storage/box/syndie_kit/syringe_gun = 0.5,
+		/obj/item/reagent_containers/syringe/drugs = 1,
+		/obj/item/reagent_containers/inhaler/space_drugs = 2,
+		/obj/item/grenade/spawnergrenade/manhacks = 0.2,
+		/obj/item/storage/box/smokes = 0.7,
+		/obj/item/grenade/napalm = 0.4,
 		/obj/item/reagent_containers/glass/beaker/vial/random/toxin = 1,
-		/obj/item/contraband/poster = 5,
 		/obj/item/material/knife/butterfly = 2,
 		/obj/item/material/butterflyblade = 3,
 		/obj/item/material/butterflyhandle = 3,
 		/obj/item/material/wirerod = 3,
 		/obj/item/melee/baton/cattleprod = 1,
 		/obj/item/material/knife/tacknife = 1,
+		/obj/item/material/knife/bayonet = 2,
+		/obj/item/device/multitool/hacktool = 0.3,
```

Comment:
similar thing as to the parapen - antags have no way to distinguish that the hack tool is not a normal multitool
don't want crew metagaming and ""randomly"" using a screwdriver on a multitool to find out it's not just a normal multitool

Reply:
Seems harmless enough to not really be a concern to me

Reply:
This sounds like an issue with the item more than anything else. I will take a look at its antag-description.

Reply:
But for now I will remove it